Fix typos in documentation and example-related code

g2hhh2ee opened the pull request:
<!--  Thanks for sending a pull request! -->

#### What this PR does / why we need it?
I have gone through the documentation on the Yorkie homepage and identified several typos in the example code and documentation.

- Fixed a typo in the helm command for AWS EKS setup. 
  - I changed `--set inngress.ingressClassName=alb` to `--set ingress.ingressClassName=alb` for accurate configuration on AWS EKS. 
  ![image](https://github.com/yorkie-team/yorkie-team.github.io/assets/57996351/07ff4934-cb98-4924-b046-3a60a3003784)

#### Any background context you want to provide?
- Also, 'https://github.com/yorkie-team/yorkie-ui.git' in the `scripts/fetch-ui.sh` file is a link that doesn't exist, so `fetch:ui` script is throwing an error. Is there a link to replace that one?
https://github.com/yorkie-team/yorkie-team.github.io/blob/b1a47e3963dbfacd6e6e1043f099b901402364e5/scripts/fetch-ui.sh#L3


#### What are the relevant tickets?
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes #

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

g2hhh2ee committed:
Fix helm command typo for AWS EKS in cluster-addons.mdx

To configure correctly in AWS EKS, a typo in the Helm command for
setting up AWS EKS needs to be fixed.

g2hhh2ee committed:
Fix typos in the yorkie homepage documentations

g2hhh2ee committed:
Fix typos related to the example codes

krapie commented on docs/self-hosted-server/aws-eks.mdx:
```diff
@@ -122,8 +122,8 @@ $ helm install yorkie-cluster yorkie-team/yorkie-cluster \
-This will install `yorkie-cluster` release in AWS EKS, which can publicly accessible via `{YOUR_API_DOMAIN_NAME}` with HTTPS
-by provising external AWS ALB in front of Yorkie cluster.
+This will install `yorkie-cluster` release in AWS EKS, which can be publicly accessible via `{YOUR_API_DOMAIN_NAME}` with HTTPS
+by providing external AWS ALB in front of Yorkie cluster.
```
This was meant for `provisioning`, so `provisioning` will be better.

krapie commented on docs/self-hosted-server/minikube.mdx:
```diff
@@ -134,15 +134,15 @@ NOTES:
 yorkie-cluster successfully installed!
 
-For next steps, follow:
+For the next steps, follow:
```
I think it will be better to keep `For next steps, follow:` for now.

krapie commented on components/Icons/Icon.tsx:
```diff
@@ -1,46 +1,46 @@
-import StarSVG from '@/public/assets/icons/icon_stars.svg';
-import CopySVG from '@/public/assets/icons/icon_copy.svg';
-import BookSVG from '@/public/assets/icons/icon_book.svg';
+import DangerSVG from '@/public/assets/icons/icon_alert_danger.svg';
```
Is there any reason for reordering imports?

g2hhh2ee replied:
Sorry, that was due to the import sort settings in my vscode. I think it got reflected on save, I'll change it back.

krapie reviewed (changes requested):
Thank you for fixing all these typos!
I left small comments below.

hackerwins reviewed (changes requested):
Thanks for your contribution. Checking the entire typo on the homepage is a good idea. I left a comment.

Can we roll back typos in `className`s? Obvious typo, but the UI repository needs to be updated as well.

g2hhh2ee commented on components/exampleView/FullView.tsx:
```diff
@@ -18,7 +18,7 @@ const ExampleContent = () => {
           </label>
         </div>
         <div className="comment_box">
-          <ol className="commnet_list">
```
> Can we roll back typos in `className`s? Obvious typo, but the UI repository needs to be updated as well.

@hackerwins

Is this what you're referring to? <ol className="commnet_list">
If so, I also fixed it in the styles/style.css file as well.

I didn't know it needed to be updated in the UI repository though.
I'll revert it back

hackerwins replied:
If you're interested in fixing typos in UI repositories, I'd be happy to grant you permission to access the UI repository. Just let me know!

g2hhh2ee replied:
I am interested in fixing typos in the UI repository, and I'd be happy to assist. If you grant me the necessary access, I'll gladly collaborate on correcting the typos. Please let me know if there are any specific permissions or instructions I need to follow.

hackerwins replied:
@g2hhh2ee I have just sent you an invitation.

g2hhh2ee committed:
Fix Typo in aws-eks

Fix typo and make the following adjustments:

- Corrected Typo:
  - Fixed the typo "provising" to "providing".
  - Final Change: Typo corrected to "provisioning".

- Wording Adjustment:
  - Adjusted "For next steps" to "For the next steps".
  - Reverted back to "For next steps" based on review.

- Reverted Icon Imports:
  - Previous Change: Unnecessarily sorted icon imports.
  - Reverted icon imports to their original order.

- Reverted class name changes:
  - Reverted class name `comment_list` to `commnet_list` due to the
    necessity of prior changes in the UI repository.

krapie reviewed (approved):
Thank you for your contribution.
We will merge this PR after we merge your PR in the UI repository.

hackerwins reviewed (approved):
Thanks for your contribution.

hackerwins commented:
@g2hhh2ee Could you fix the conflict?

g2hhh2ee commented:
Sure! Thanks for letting me know!

g2hhh2ee committed:
Merge branch 'main' into fix/typo
